BraveShared/Analytics/DAU.swift

```diff
@@ -38,6 +40,11 @@ public class DAU {
             return false
         }
         
+        if processingPing {
```

Comment:
This is redundant once you make the DAU class object global. The check below for launchTimer != nil will make sure no new timers are created. Instead put this check at `guard let paramsAndPrefs = paramsAndPrefsSetup() else ` and set `processingPing = true` after the guard

Reply:
Did move, and `dau.sendPingToServer()` is still called manually on app open.